syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendsRequest{
    uint32 userid = 1;
}

message GetFriendsResponse{
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendsServiceRpc{
    rpc GetFriendsList(GetFriendsRequest) returns(GetFriendsResponse);
}